dnsimple/webhook/events.go

```diff
@@ -125,6 +128,21 @@ func (d *AccountMembershipEventData) unmarshalEventData(payload []byte) error {
 	return unmarshalEventData(payload, d)
 }
 
+//
+// SsoEvent
+//
+
+// SsoEventData represents the data node for an single sign-on (SSO) event.
+type SsoEventData struct {
```

Comment:
Given the other `account.*` events use the `Account` prefix, does it make sense to do the same here (eg. `AccountSsoEventData`)?

Reply:
Done https://github.com/dnsimple/dnsimple-go/pull/227/commits/f3b7c10f751d6ba266cedf177a3918b816eab7a8